cv32e40p/sim/ExternalRepos.mk

```diff
@@ -14,8 +14,8 @@
 export SHELL = /bin/bash
 
 CV_CORE_REPO   ?= https://github.com/openhwgroup/cv32e40p
-CV_CORE_BRANCH ?= master
-CV_CORE_HASH   ?= d0d1c25
+CV_CORE_BRANCH ?= dev
+CV_CORE_HASH   ?= bddbd38fb0b5ced6208090691002b7ee021dcc8d
```

Comment:
This branch will not compile with `dsim`, see [cv32e40p #719](https://github.com/openhwgroup/cv32e40p/pull/719).